implementations/rust/ockam/ockam_examples/example_projects/tunnel/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "tunnel"
+version = "0.0.0"
+authors = ["Ockam Developers"]
+edition = "2018"
+
+[dependencies]
+ockam = { path = "../../../ockam" }
+nanorand = "0.6"
```

Comment:
Nit: If pressed I probably prefer `nanorand`, but we use `rand` everywhere else, so it's probably worth being consistent.